src/index.js

```diff
@@ -1,10 +1,34 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
 import App from './components/App';
 
+import booksReducer from './reducers/books';
+
+const initialState = [{
+  id: Math.round(Math.random() * (100 - 1) + 1),
```

Comment:
- [ ] Please **never use a random number to generate ids.**
  - That's very unsafe and can cause multiple bugs and logic errors in case the ids are not unique, which can happen if you use a random function.
  - If you want to have ids please use a proper npm package made for generating unique ids, such as [uuid](https://www.npmjs.com/package/uuid) 
  - Or implement your own id counter system, you can have a `nextId` key in your store, increment it every time a new book is created and use that value from the store for the created book id.